policyengine_us/parameters/gov/states/co/ssa/oap/age_range.yaml

```
@@ -0,0 +1,18 @@
+description: Age range defining someone eligible for OAP
```

Similar phrasing for other descriptions
```suggestion
description: Colorado limits its Old Age Pension to individuals in this age range.
```